md/automating-builds.md

```diff
@@ -4,6 +4,11 @@
 **Note:** For Enterprise, Performance, Efficiency, and Teamwork editions only.
 :::
 
+::: warning
+**Note:**
+* The solution described here relies on the BonitaStudioBuilder which is deprerated as of Bonita 7.7
```

Comment:
```suggestion
* The solution described here relies on the BonitaStudioBuilder, which has been deprecated since Bonita 7.7.0. Instead, we strongly encourage you to use the LA builder included in the tooling suite of Bonita Continuous Delivery add-on. One added-value is that LA builder does not need a studio to be installed.
```